plugins/woo-ai/package.json

```diff
@@ -125,6 +125,27 @@
 		"watch:build:project:wp-scripts": {
 			"command": "wp-scripts start",
 			"service": true
+		},
+		"dependencyOutputs": {
```

Comment:
These are generated postinstall, so I wonder if it wasn't checked in for some reason.